src/main/resources/application.yaml

```diff
@@ -1,11 +1,14 @@
-resilience4j.retry:
-  configs:
-    default:
-      maxAttempts: 3
-      waitDuration: 100ms
-      enableExponentialBackoff: true
-      exponentialBackoffMultiplier: 2
-      retryExceptions:
-        - org.springframework.web.client.HttpServerErrorException
-        - java.io.IOException
-      ignoreExceptions: []
+token:
+  secret-key: 8692tg18dEq2twmv48AtiKWp3Qo/2nw6/tU6Rzttm9Q=
```

Comment:
move that to application-local please

Reply:
tests don't run after moving this line to application-local as they also use this token

Reply:
i set off gitguardian